### ISSUE TITLE
*** Error in `./bitcoin-qt': corrupted double-linked list: 0x00007f53acb903a0 ***

### ISSUE THREAD

**Issue by soeren-b-c** [gandrewstone/bitcoin#168]
My bitcoin QT  v0.12.1.0-bd24b82 (64-bit) crashed on Ubuntu. This was what I got from the console:

It hasn't happened since (so far...)
Let me know if there is any other info you need.

*** Error in `./bitcoin-qt': corrupted double-linked list: 0x00007f53acb903a0 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7f53b882a7e5]
/lib/x86_64-linux-gnu/libc.so.6(+0x7e6f8)[0x7f53b88316f8]
/lib/x86_64-linux-gnu/libc.so.6(+0x800a8)[0x7f53b88330a8]
/lib/x86_64-linux-gnu/libc.so.6(cfree+0x4c)[0x7f53b883698c]
./bitcoin-qt(+0x32f56f)[0x55a2a3f9956f]
/lib/x86_64-linux-gnu/libc.so.6(+0x39ff8)[0x7f53b87ecff8]
/lib/x86_64-linux-gnu/libc.so.6(+0x3a045)[0x7f53b87ed045]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf7)[0x7f53b87d3837]
./bitcoin-qt(+0x1675b7)[0x55a2a3dd15b7]
======= Memory map: ========
55a2a3c6a000-55a2a5b27000 r-xp 00000000 08:01 26217434                   /home/bitc/bitcoinUnlimited-0.12.1/bin/bitcoin-qt
55a2a5d26000-55a2a5db7000 r--p 01ebc000 08:01 26217434                   /home/bitc/bitcoinUnlimited-0.12.1/bin/bitcoin-qt
55a2a5db7000-55a2a5dcb000 rw-p 01f4d000 08:01 26217434                   /home/bitc/bitcoinUnlimited-0.12.1/bin/bitcoin-qt
55a2a5dcb000-55a2a5eb4000 rw-p 00000000 00:00 0 
55a2a714e000-55a2a85c7000 rw-p 00000000 00:00 0                          [heap]
7f533c000000-7f533fffa000 rw-p 00000000 00:00 0 
7f533fffa000-7f5340000000 ---p 00000000 00:00 0 
7f5344000000-7f5347feb000 rw-p 00000000 00:00 0 
7f5347feb000-7f5348000000 ---p 00000000 00:00 0 
7f5348000000-7f5348021000 rw-p 00000000 00:00 0 
7f5348021000-7f534c000000 ---p 00000000 00:00 0 
7f5350000000-7f5350021000 rw-p 00000000 00:00 0 
7f5350021000-7f5354000000 ---p 00000000 00:00 0 
7f5354000000-7f5354021000 rw-p 00000000 00:00 0 
7f5354021000-7f5358000000 ---p 00000000 00:00 0 
7f5358000000-7f535c000000 rw-p 00000000 00:00 0 
7f535c000000-7f535c021000 rw-p 00000000 00:00 0 
7f535c021000-7f5360000000 ---p 00000000 00:00 0 
7f5360000000-7f5360021000 rw-p 00000000 00:00 0 
7f5360021000-7f5364000000 ---p 00000000 00:00 0 
7f5364000000-7f5364639000 rw-p 00000000 00:00 0 
7f5364639000-7f5368000000 ---p 00000000 00:00 0 
7f5368000000-7f536808d000 rw-p 00000000 00:00 0 
7f536808d000-7f536c000000 ---p 00000000 00:00 0 
7f536dffc000-7f536dffd000 ---p 00000000 00:00 0 
7f536dffd000-7f536e7fd000 rw-p 00000000 00:00 0 
7f5370000000-7f5370021000 rw-p 00000000 00:00 0 
7f5370021000-7f5374000000 ---p 00000000 00:00 0 
7f5374000000-7f5375641000 rw-p 00000000 00:00 0 
7f5375641000-7f5378000000 ---p 00000000 00:00 0 
7f5378000000-7f5378021000 rw-p 00000000 00:00 0 
7f5378021000-7f537c000000 ---p 00000000 00:00 0 
7f5380000000-7f53800e3000 rw-p 00000000 00:00 0 
7f53800e3000-7f5384000000 ---p 00000000 00:00 0 
7f5384000000-7f538716f000 rw-p 00000000 00:00 0 
7f538716f000-7f5388000000 ---p 00000000 00:00 0 
7f5388000000-7f5388023000 rw-p 00000000 00:00 0 
7f5388023000-7f538c000000 ---p 00000000 00:00 0 
7f538c000000-7f5390000000 rw-p 00000000 00:00 0 
7f5390000000-7f53903e1000 rw-p 00000000 00:00 0 
7f53903e1000-7f5394000000 ---p 00000000 00:00 0 
7f5394282000-7f5394299000 r-xp 00000000 08:01 15205725                   /lib/x86_64-linux-gnu/libresolv-2.23.so
7f5394299000-7f5394499000 ---p 00017000 08:01 15205725                   /lib/x86_64-linux-gnu/libresolv-2.23.so
7f5394499000-7f539449a000 r--p 00017000 08:01 15205725                   /lib/x86_64-linux-gnu/libresolv-2.23.so
7f539449a000-7f539449b000 rw-p 00018000 08:01 15205725                   /lib/x86_64-linux-gnu/libresolv-2.23.so
7f539449b000-7f539449d000 rw-p 00000000 00:00 0 
7f5395917000-7f5395922000 r-xp 00000000 08:01 15205724                   /lib/x86_64-linux-gnu/libnss_files-2.23.so
7f5395922000-7f5395b21000 ---p 0000b000 08:01 15205724                   /lib/x86_64-linux-gnu/libnss_files-2.23.so
7f5395b21000-7f5395b22000 r--p 0000a000 08:01 15205724                   /lib/x86_64-linux-gnu/libnss_files-2.23.so
7f5395b22000-7f5395b23000 rw-p 0000b000 08:01 15205724                   /lib/x86_64-linux-gnu/libnss_files-2.23.so
7f5395b23000-7f5395b29000 rw-p 00000000 00:00 0 
7f5398000000-7f5398023000 rw-p 00000000 00:00 0 
7f5398023000-7f539c000000 ---p 00000000 00:00 0 
7f539cf78000-7f539cf79000 ---p 00000000 00:00 0 
7f539cf79000-7f539d779000 rw-p 00000000 00:00 0 
7f539d77c000-7f539d781000 r-xp 00000000 08:01 15205723                   /lib/x86_64-linux-gnu/libnss_dns-2.23.so
7f539d781000-7f539d981000 ---p 00005000 08:01 15205723                   /lib/x86_64-linux-gnu/libnss_dns-2.23.so
7f539d981000-7f539d982000 r--p 00005000 08:01 15205723                   /lib/x86_64-linux-gnu/libnss_dns-2.23.so
7f539d982000-7f539d983000 rw-p 00006000 08:01 15205723                   /lib/x86_64-linux-gnu/libnss_dns-2.23.so
7f539dda1000-7f539dda3000 r-xp 00000000 08:01 15208898                   /lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2
7f539dda3000-7f539dfa2000 ---p 00002000 08:01 15208898                   /lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2
7f539dfa2000-7f539dfa3000 r--p 00001000 08:01 15208898                   /lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2
7f539dfa3000-7f539dfa4000 rw-p 00002000 08:01 15208898                   /lib/x86_64-linux-gnu/libnss_mdns4_minimal.so.2
7f53a0000000-7f53a0023000 rw-p 00000000 00:00 0 
7f53a0023000-7f53a4000000 ---p 00000000 00:00 0 
7f53a8000000-7f53a8021000 rw-p 00000000 00:00 0 
7f53a8021000-7f53ac000000 ---p 00000000 00:00 0 
7f53ac000000-7f53b0000000 rw-p 00000000 00:00 0 
7f53b0000000-7f53b0021000 rw-p 00000000 00:00 0 
7f53b0021000-7f53b4000000 ---p 00000000 00:00 0 
7f53b4a46000-7f53b4a47000 ---p 00000000 00:00 0 
7f53b4a47000-7f53b5247000 rw-p 00000000 00:00 0 
7f53b529b000-7f53b529c000 ---p 00000000 00:00 0 
7f53b529c000-7f53b5a9c000 rw-p 00000000 00:00 0 
7f53b6ad0000-7f53b6ad5000 r-xp 00000000 08:01 25173681                   /usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
7f53b6ad5000-7f53b6cd4000 ---p 00005000 08:01 25173681                   /usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
7f53b6cd4000-7f53b6cd5000 r--p 00004000 08:01 25173681                   /usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
7f53b6cd5000-7f53b6cd6000 rw-p 00005000 08:01 25173681                   /usr/lib/x86_64-linux-gnu/libXfixes.so.3.1.0
7f53b6cd6000-7f53b6cdf000 r-xp 00000000 08:01 25173699                   /usr/lib/x86_64-linux-gnu/libXrender.so.1.3.0
7f53b6cdf000-7f53b6ede000 ---p 00009000 08:01 25173699                   /usr/lib/x86_64-linux-gnu/libXrender.so.1.3.0
7f53b6ede000-7f53b6edf000 r--p 00008000 08:01 25173699                   /usr/lib/x86_64-linux-gnu/libXrender.so.1.3.0
7f53b6edf000-7f53b6ee0000 rw-p 00009000 08:01 25173699                   /usr/lib/x86_64-linux-gnu/libXrender.so.1.3.0
7f53b6ee0000-7f53b6ee9000 r-xp 00000000 08:01 25173673                   /usr/lib/x86_64-linux-gnu/libXcursor.so.1.0.2
7f53b6ee9000-7f53b70e8000 ---p 00009000 08:01 25173673                   /usr/lib/x86_64-linux-gnu/libXcursor.so.1.0.2
7f53b70e8000-7f53b70e9000 r--p 00008000 08:01 25173673                   /usr/lib/x86_64-linux-gnu/libXcursor.so.1.0.2
7f53b70e9000-7f53b70ea000 rw-p 00009000 08:01 25173673                   /usr/lib/x86_64-linux-gnu/libXcursor.so.1.0.2
7f53b70ea000-7f53b70eb000 ---p 00000000 00:00 0 
7f53b70eb000-7f53b78eb000 rw-p 00000000 00:00 0 
7f53b78eb000-7f53b7d41000 r--p 00000000 08:01 25168314                   /usr/lib/locale/locale-archive
7f53b7d41000-7f53b7d65000 r-xp 00000000 08:01 15208934                   /lib/x86_64-linux-gnu/libpng12.so.0.54.0
7f53b7d65000-7f53b7f64000 ---p 00024000 08:01 15208934                   /lib/x86_64-linux-gnu/libpng12.so.0.54.0
7f53b7f64000-7f53b7f65000 r--p 00023000 08:01 15208934                   /lib/x86_64-linux-gnu/libpng12.so.0.54.0
7f53b7f65000-7f53b7f66000 rw-p 00024000 08:01 15208934                   /lib/x86_64-linux-gnu/libpng12.so.0.54.0
7f53b7f66000-7f53b7f7f000 r-xp 00000000 08:01 15208984                   /lib/x86_64-linux-gnu/libz.so.1.2.8
7f53b7f7f000-7f53b817e000 ---p 00019000 08:01 15208984                   /lib/x86_64-linux-gnu/libz.so.1.2.8
7f53b817e000-7f53b817f000 r--p 00018000 08:01 15208984                   /lib/x86_64-linux-gnu/libz.so.1.2.8
7f53b817f000-7f53b8180000 rw-p 00019000 08:01 15208984                   /lib/x86_64-linux-gnu/libz.so.1.2.8
7f53b8180000-7f53b81a6000 r-xp 00000000 08:01 15204523                   /lib/x86_64-linux-gnu/libexpat.so.1.6.0
7f53b81a6000-7f53b83a6000 ---p 00026000 08:01 15204523                   /lib/x86_64-linux-gnu/libexpat.so.1.6.0
7f53b83a6000-7f53b83a8000 r--p 00026000 08:01 15204523                   /lib/x86_64-linux-gnu/libexpat.so.1.6.0
7f53b83a8000-7f53b83a9000 rw-p 00028000 08:01 15204523                   /lib/x86_64-linux-gnu/libexpat.so.1.6.0
7f53b83a9000-7f53b83ae000 r-xp 00000000 08:01 25173677                   /usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
7f53b83ae000-7f53b85ad000 ---p 00005000 08:01 25173677                   /usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
7f53b85ad000-7f53b85ae000 r--p 00004000 08:01 25173677                   /usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
7f53b85ae000-7f53b85af000 rw-p 00005000 08:01 25173677                   /usr/lib/x86_64-linux-gnu/libXdmcp.so.6.0.0
7f53b85af000-7f53b85b1000 r-xp 00000000 08:01 25173666                   /usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
7f53b85b1000-7f53b87b1000 ---p 00002000 08:01 25173666                   /usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
7f53b87b1000-7f53b87b2000 r--p 00002000 08:01 25173666                   /usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
7f53b87b2000-7f53b87b3000 rw-p 00003000 08:01 25173666                   /usr/lib/x86_64-linux-gnu/libXau.so.6.0.0
7f53b87b3000-7f53b8972000 r-xp 00000000 08:01 15205642                   /lib/x86_64-linux-gnu/libc-2.23.so
7f53b8972000-7f53b8b72000 ---p 001bf000 08:01 15205642                   /lib/x86_64-linux-gnu/libc-2.23.so
7f53b8b72000-7f53b8b76000 r--p 001bf000 08:01 15205642                   /lib/x86_64-linux-gnu/libc-2.23.so
7f53b8b76000-7f53b8b78000 rw-p 001c3000 08:01 15205642                   /lib/x86_64-linux-gnu/libc-2.23.so
7f53b8b78000-7f53b8b7c000 rw-p 00000000 00:00 0 
7f53b8b7c000-7f53b8b92000 r-xp 00000000 08:01 15204366                   /lib/x86_64-linux-gnu/libgcc_s.so.1
7f53b8b92000-7f53b8d91000 ---p 00016000 08:01 15204366                   /lib/x86_64-linux-gnu/libgcc_s.so.1
7f53b8d91000-7f53b8d92000 rw-p 00015000 08:01 15204366                   /lib/x86_64-linux-gnu/libgcc_s.so.1
7f53b8d92000-7f53b8e9a000 r-xp 00000000 08:01 15205718                   /lib/x86_64-linux-gnu/libm-2.23.so
7f53b8e9a000-7f53b9099000 ---p 00108000 08:01 15205718                   /lib/x86_64-linux-gnu/libm-2.23.so
7f53b9099000-7f53b909a000 r--p 00107000 08:01 15205718                   /lib/x86_64-linux-gnu/libm-2.23.so
7f53b909a000-7f53b909b000 rw-p 00108000 08:01 15205718                   /lib/x86_64-linux-gnu/libm-2.23.so
7f53b909b000-7f53b90a2000 r-xp 00000000 08:01 15204627                   /lib/x86_64-linux-gnu/librt-2.23.so
7f53b90a2000-7f53b92a1000 ---p 00007000 08:01 15204627                   /lib/x86_64-linux-gnu/librt-2.23.so
7f53b92a1000-7f53b92a2000 r--p 00006000 08:01 15204627                   /lib/x86_64-linux-gnu/librt-2.23.so
7f53b92a2000-7f53b92a3000 rw-p 00007000 08:01 15204627                   /lib/x86_64-linux-gnu/librt-2.23.so
7f53b92a3000-7f53b92a6000 r-xp 00000000 08:01 15204631                   /lib/x86_64-linux-gnu/libanl-2.23.so
7f53b92a6000-7f53b94a5000 ---p 00003000 08:01 15204631                   /lib/x86_64-linux-gnu/libanl-2.23.so
7f53b94a5000-7f53b94a6000 r--p 00002000 08:01 15204631                   /lib/x86_64-linux-gnu/libanl-2.23.so
7f53b94a6000-7f53b94a7000 rw-p 00003000 08:01 15204631                   /lib/x86_64-linux-gnu/libanl-2.23.so
7f53b94a7000-7f53b94aa000 r-xp 00000000 08:01 15205638                   /lib/x86_64-linux-gnu/libdl-2.23.so
7f53b94aa000-7f53b96a9000 ---p 00003000 08:01 15205638                   /lib/x86_64-linux-gnu/libdl-2.23.so
7f53b96a9000-7f53b96aa000 r--p 00002000 08:01 15205638                   /lib/x86_64-linux-gnu/libdl-2.23.so
7f53b96aa000-7f53b96ab000 rw-p 00003000 08:01 15205638                   /lib/x86_64-linux-gnu/libdl-2.23.so
7f53b96ab000-7f53b96c3000 r-xp 00000000 08:01 15205719                   /lib/x86_64-linux-gnu/libpthread-2.23.so
7f53b96c3000-7f53b98c2000 ---p 00018000 08:01 15205719                   /lib/x86_64-linux-gnu/libpthread-2.23.so
7f53b98c2000-7f53b98c3000 r--p 00017000 08:01 15205719                   /lib/x86_64-linux-gnu/libpthread-2.23.so
7f53b98c3000-7f53b98c4000 rw-p 00018000 08:01 15205719                   /lib/x86_64-linux-gnu/libpthread-2.23.so
7f53b98c4000-7f53b98c8000 rw-p 00000000 00:00 0 
7f53b98c8000-7f53b996c000 r-xp 00000000 08:01 25170959                   /usr/lib/x86_64-linux-gnu/libfreetype.so.6.12.1
7f53b996c000-7f53b9b6b000 ---p 000a4000 08:01 25170959                   /usr/lib/x86_64-linux-gnu/libfreetype.so.6.12.1
7f53b9b6b000-7f53b9b71000 r--p 000a3000 08:01 25170959                   /usr/lib/x86_64-linux-gnu/libfreetype.so.6.12.1
7f53b9b71000-7f53b9b72000 rw-p 000a9000 08:01 25170959                   /usr/lib/x86_64-linux-gnu/libfreetype.so.6.12.1
7f53b9b72000-7f53b9baf000 r-xp 00000000 08:01 25165974                   /usr/lib/x86_64-linux-gnu/libfontconfig.so.1.9.0
7f53b9baf000-7f53b9dae000 ---p 0003d000 08:01 25165974                   /usr/lib/x86_64-linux-gnu/libfontconfig.so.1.9.0
7f53b9dae000-7f53b9db0000 r--p 0003c000 08:01 25165974                   /usr/lib/x86_64-linux-gnu/libfontconfig.so.1.9.0
7f53b9db0000-7f53b9db5000 rw-p 0003e000 08:01 25165974                   /usr/lib/x86_64-linux-gnu/libfontconfig.so.1.9.0
7f53b9db5000-7f53b9dd6000 r-xp 00000000 08:01 25175033                   /usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
7f53b9dd6000-7f53b9fd5000 ---p 00021000 08:01 25175033                   /usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
7f53b9fd5000-7f53b9fd6000 r--p 00020000 08:01 25175033                   /usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
7f53b9fd6000-7f53b9fd7000 rw-p 00021000 08:01 25175033                   /usr/lib/x86_64-linux-gnu/libxcb.so.1.1.0
7f53b9fd7000-7f53ba10c000 r-xp 00000000 08:01 25173662                   /usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
7f53ba10c000-7f53ba30c000 ---p 00135000 08:01 25173662                   /usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
7f53ba30c000-7f53ba30d000 r--p 00135000 08:01 25173662                   /usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
7f53ba30d000-7f53ba311000 rw-p 00136000 08:01 25173662                   /usr/lib/x86_64-linux-gnu/libX11.so.6.3.0
7f53ba311000-7f53ba312000 r-xp 00000000 08:01 25173660                   /usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
7f53ba312000-7f53ba511000 ---p 00001000 08:01 25173660                   /usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
7f53ba511000-7f53ba512000 r--p 00000000 08:01 25173660                   /usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
7f53ba512000-7f53ba513000 rw-p 00001000 08:01 25173660                   /usr/lib/x86_64-linux-gnu/libX11-xcb.so.1.0.0
7f53ba513000-7f53ba539000 r-xp 00000000 08:01 15204569                   /lib/x86_64-linux-gnu/ld-2.23.so
7f53ba5e4000-7f53ba5e5000 rw-p 00000000 00:00 0 
7f53ba5e5000-7f53ba5e9000 r--s 00000000 08:01 12856295                   /var/cache/fontconfig/6aa41aa22e18b8fa06a12da28ea9c28b-le64.cache-6
7f53ba5e9000-7f53ba5f4000 r--s 00000000 08:01 12856294                   /var/cache/fontconfig/945677eb7aeaf62f1d50efc3fb3ec7d8-le64.cache-6
7f53ba5f4000-7f53ba5f6000 r--s 00000000 08:01 12856288                   /var/cache/fontconfig/99e8ed0e538f840c565b6ed5dad60d56-le64.cache-6
7f53ba5f6000-7f53ba5ff000 r--s 00000000 08:01 12856287                   /var/cache/fontconfig/a6d8cf8e4ec09cdbc8633c31745a07dd-le64.cache-6
7f53ba5ff000-7f53ba605000 r--s 00000000 08:01 12856285                   /var/cache/fontconfig/2cd17615ca594fa2959ae173292e504c-le64.cache-6
7f53ba605000-7f53ba606000 r--s 00000000 08:01 12856284                   /var/cache/fontconfig/0d8c3b2ac0904cb8a57a757ad11a4a08-le64.cache-6
7f53ba606000-7f53ba61b000 r--s 00000000 08:01 12856283                   /var/cache/fontconfig/04aabc0a78ac019cf9454389977116d2-le64.cache-6
7f53ba61b000-7f53ba61c000 r--s 00000000 08:01 12856282                   /var/cache/fontconfig/1ac9eb803944fde146138c791f5cc56a-le64.cache-6
7f53ba61c000-7f53ba620000 r--s 00000000 08:01 12856281                   /var/cache/fontconfig/385c0604a188198f04d133e54aba7fe7-le64.cache-6
7f53ba620000-7f53ba621000 r--s 00000000 08:01 12856280                   /var/cache/fontconfig/dc05db6664285cc2f12bf69c139ae4c3-le64.cache-6
7f53ba621000-7f53ba624000 r--s 00000000 08:01 12856279                   /var/cache/fontconfig/767a8244fc0220cfb567a839d0392e0b-le64.cache-6
7f53ba624000-7f53ba625000 r--s 00000000 08:01 12856260                   /var/cache/fontconfig/4794a0821666d79190d59a36cb4f44b5-le64.cache-6
7f53ba625000-7f53ba62a000 r--s 00000000 08:01 12856259                   /var/cache/fontconfig/8801497958630a81b71ace7c5f9b32a8-le64.cache-6
7f53ba62a000-7f53ba665000 r--s 00000000 08:01 27009524                   /home/bitc/.cache/fontconfig/365b55f210c0a22e9a19e35191240f32-le64.cache-6
7f53ba665000-7f53ba666000 r--s 00000000 08:01 12856254                   /var/cache/fontconfig/bab58bb527bb656aaa9f116d68a48d89-le64.cache-6
7f53ba666000-7f53ba66d000 r--s 00000000 08:01 12856250                   /var/cache/fontconfig/3047814df9a2f067bd2d96a2b9c36e5a-le64.cache-6
7f53ba66d000-7f53ba675000 r--s 00000000 08:01 12856245                   /var/cache/fontconfig/bf3b770c553c462765856025a94f1ce6-le64.cache-6
7f53ba675000-7f53ba676000 r--s 00000000 08:01 12856244                   /var/cache/fontconfig/56cf4f4769d0f4abc89a4895d7bd3ae1-le64.cache-6
7f53ba676000-7f53ba677000 r--s 00000000 08:01 12856243                   /var/cache/fontconfig/b9d506c9ac06c20b433354fa67a72993-le64.cache-6
7f53ba677000-7f53ba67d000 r--s 00000000 08:01 12856242                   /var/cache/fontconfig/b47c4e1ecd0709278f4910c18777a504-le64.cache-6
7f53ba67d000-7f53ba685000 r--s 00000000 08:01 12856241                   /var/cache/fontconfig/52f7bdb7ce746bfd7eaa1985bd9cfa93-le64.cache-6
7f53ba685000-7f53ba698000 r--s 00000000 08:01 12856240                   /var/cache/fontconfig/d52a8644073d54c13679302ca1180695-le64.cache-6
7f53ba698000-7f53ba699000 r--s 00000000 08:01 12856239                   /var/cache/fontconfig/551ecf3b0e8b0bca0f25c0944f561853-le64.cache-6
7f53ba699000-7f53ba6a4000 r--s 00000000 08:01 12856238                   /var/cache/fontconfig/d589a48862398ed80a3d6066f4f56f4c-le64.cache-6
7f53ba6a4000-7f53ba6a8000 r--s 00000000 08:01 12856237                   /var/cache/fontconfig/246184dc75a16901ca37d96895904249-le64.cache-6
7f53ba6a8000-7f53ba6a9000 r--s 00000000 08:01 12856236                   /var/cache/fontconfig/30829fa25452a46451e813d634d7f916-le64.cache-6
7f53ba6a9000-7f53ba6aa000 r--s 00000000 08:01 12856235                   /var/cache/fontconfig/0c9eb80ebd1c36541ebe2852d3bb0c49-le64.cache-6
7f53ba6aa000-7f53ba6ad000 r--s 00000000 08:01 12856233                   /var/cache/fontconfig/75114ca45c98e8a441da0ff356701271-le64.cache-6
7f53ba6ad000-7f53ba6b8000 r--s 00000000 08:01 12856232                   /var/cache/fontconfig/83bf95040141907cd45bb53cf7c1c148-le64.cache-6
7f53ba6b8000-7f53ba6ca000 r--s 00000000 08:01 12856231                   /var/cache/fontconfig/9b89f8e3dae116d678bbf48e5f21f69b-le64.cache-6
7f53ba6ca000-7f53ba6ce000 r--s 00000000 08:01 12856230                   /var/cache/fontconfig/6c08beecf0dac481ec92e759e0c2e6d7-le64.cache-6
7f53ba6ce000-7f53ba6d2000 r--s 00000000 08:01 12856229                   /var/cache/fontconfig/4d6aee6d44eccb37054d3216e945f618-le64.cache-6
7f53ba6d2000-7f53ba6d6000 r--s 00000000 08:01 12856228                   /var/cache/fontconfig/d6c6df98c3df5faf9bd0f8ef4d91efe5-le64.cache-6
7f53ba6d6000-7f53ba6da000 r--s 00000000 08:01 12855797                   /var/cache/fontconfig/9fb319ef52839f11ee26ab8b4cffa12b-le64.cache-6
7f53ba6da000-7f53ba6f0000 r--s 00000000 08:01 12856226                   /var/cache/fontconfig/e13b20fdb08344e0e664864cc2ede53d-le64.cache-6
7f53ba6f0000-7f53ba6f5000 r--s 00000000 08:01 12848600                   /var/cache/fontconfig/7ef2298fde41cc6eeb7af42e48b7d293-le64.cache-6
7f53ba6f5000-7f53ba6f7000 r--s 00000000 08:01 12856299                   /var/cache/fontconfig/16326683038b281783a0ef8c680e3a10-le64.cache-6
7f53ba6f7000-7f53ba704000 r--s 00000000 08:01 12856298                   /var/cache/fontconfig/8f02d4cb045bd6ce15663e43f347c9f8-le64.cache-6
7f53ba704000-7f53ba705000 r--s 00000000 08:01 12856297                   /var/cache/fontconfig/e0aa53bcfa504e64f87823c16bc01eb6-le64.cache-6
7f53ba705000-7f53ba70e000 rw-p 00000000 00:00 0 
7f53ba70e000-7f53ba72f000 r--s 00000000 08:01 12850286                   /var/cache/fontconfig/467c019e582ee353435ea5c21d137ef6-le64.cache-6
7f53ba72f000-7f53ba736000 r--s 00000000 08:01 25563725                   /usr/lib/x86_64-linux-gnu/gconv/gconv-modules.cache
7f53ba736000-7f53ba738000 rw-p 00000000 00:00 0 
7f53ba738000-7f53ba739000 r--p 00025000 08:01 15204569                   /lib/x86_64-linux-gnu/ld-2.23.so
7f53ba739000-7f53ba73a000 rw-p 00026000 08:01 15204569                   /lib/x86_64-linux-gnu/ld-2.23.so
7f53ba73a000-7f53ba73b000 rw-p 00000000 00:00 0 
7ffe46c8c000-7ffe46cad000 rw-p 00000000 00:00 0                          [stack]
7ffe46cf1000-7ffe46cf3000 r--p 00000000 00:00 0                          [vvar]
7ffe46cf3000-7ffe46cf5000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]

**Comment by sickpig** [comment]
@soeren-b-c 

try to use our nightly ubuntu repo since it provides a more recent version of the software with a bunch of fixes that could solve your problem: 

```
sudo apt-get install software-properties-common
sudo add-apt-repository ppa:bitcoin-unlimited/bu-ppa-nightly
sudo apt-get update
sudo apt-get install bitcoind bitcoin-qt
```

**Comment by sickpig** [comment]
@gandrewstone @AndrewClifford I think we could close this one

**Comment by sickpig** [comment]
Close due to lack of feedback and probably fixed by #138
